ledger/participant-integration-api/src/main/scala/platform/index/InMemoryStateUpdater.scala

```diff
@@ -93,15 +93,15 @@ private[platform] object InMemoryStateUpdater {
       InstrumentedExecutors.newWorkStealingExecutor(
         metrics.daml.lapi.threadpool.indexBypass.prepareUpdates,
         prepareUpdatesParallelism,
-        metrics.registry,
+        metrics.dropwizardFactory.registry,
```

Comment:
All this code is still dependent on Dropwizard.
Not sure how to resolve that. Maybe with the in-memory factory, but I don't know if it makes sense in this context.
Likely something to be recorded in a ticket somewhere.

Reply:
As part of the switch to opentelemetry this should also be solved.